src/Domain/Model/Restaurant.php

```diff
@@ -0,0 +1,219 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Domain\Model;
+
+use App\Domain\Migrations\TablesNameCatalog;
+use App\Domain\Validator\PhoneNumber;
+use DateTimeImmutable;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Uid\Uuid;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name=TablesNameCatalog::RESTAURANTS)
+ */
+class Restaurant
+{
+    /**
+     * @ORM\Id()
+     * @ORM\Column(type="string", unique=true)
+     * @Assert\Uuid
+     */
+    private Uuid $id;
+
+    /**
+     * @ORM\Column(type="string", length=150)
+     * @Assert\NotBlank()
+     */
+    private string $name;
+
+    /**
+     * @ORM\Column(type="string", length=150, unique=true)
+     * @Assert\NotBlank()
+     */
+    private string $alias;
+
+    /**
+     * @ORM\Column(type="string", length=255, nullable=true)
+     * @Assert\Url
+     */
+    private ?string $url;
+
+    /**
+     * @ORM\Column(type="array", nullable=true)
+     */
+    private ?array $categories = [];
+
+    /**
+     * @ORM\Column(type="string", length=150, nullable=true)
+     */
+    private ?string $location;
+
+    /**
+     * @ORM\Column(type="string", length=150, nullable=true)
+     */
+    private ?string $coordinates;
+
```

Comment:
```suggestion

```
I think we can remove this line, since we don't add it in migration. We'll add this later in other PR.